spec/pushmi_pullyu/cli_spec.rb

```diff
@@ -200,7 +200,7 @@
       end
 
       it 'results in an identical options hash, except for config_file' do
-        cli.parse
+        cli.parse([])
```

Comment:
Test was failing because it was using ARGV which in this case belongs to rspec and thus passing in rspec args: 
```
["--pattern", "spec/**{,/*/**}/*_spec.rb"]
```
Since this isn't a proper option for PushmiPullyu it fails. Instead let's just stub args here as an empty array.